include/mbedtls/pk_internal.h

```diff
@@ -82,7 +82,10 @@ struct mbedtls_pk_info_t
     /** Interface with the debug module */
     void (*debug_func)( const void *ctx, mbedtls_pk_debug_item *items );
 
+    /** Signature size */
```

Comment:
Could you clarify please?

Reply:
In this commit, if `signature_size_func` is null then `mbedtls_pk_signature_size` returns the key size.

But the behavior has changed since that commit. `signature_size_func` is now mandatory if the engine implements signature.